src/post/api/postClient/postClient.ts

```diff
@@ -0,0 +1,47 @@
+import { Post, PostDto } from "../../types";
+import { PostClientStructure } from "./types";
+
+export class PostClient implements PostClientStructure {
+  getPost = async (): Promise<Post[]> => {
```

Comment:
¿Por qué arrow function?

Reply:
Como todas las funciones hasta ahora, no veo que haya distinción con la declaración de función ya que no está usando `this`, no?

Reply:
No todas las funciones hasta ahora las escribimos con arrow function, no.

Reply:
Corregido, pero no se por qué no es una arrow function 🤔.